services-js/commissions-app/src/client/common/TextInput.stories.tsx

```diff
@@ -4,13 +4,36 @@ import TextInput from './TextInput';
 
 storiesOf('TextInput', module)
   .add('default', () => (
-    <TextInput title="First Name" name="firstName" placeholder="First Name" />
+    <TextInput
+      title="Zip Code"
+      name="zip"
+      placeholder="Zip Code"
+      value=""
+      onChange=""
+      onBlur=""
+    />
   ))
   .add('required', () => (
     <TextInput
-      title="First Name"
-      name="firstName"
-      placeholder="First Name"
+      title="Zip Code"
+      name="zip"
+      placeholder="Zip Code"
+      required
+      error="Please Enter The Correct Zip Code, Zip Codes Contains 5 Digits."
```

Comment:
Remove the `error` from this story, since it’s focusing on what `required` does.